AppCenterDataStorage/AppCenterDataStorage/Internal/LocalStore/MSDBDocumentStore.m

```diff
@@ -63,6 +66,57 @@ - (BOOL)deleteUserStorageWithAccountId:(NSString *)accountId {
   return [self.dbStorage dropTable:tableName];
 }
 
+- (MSDocumentWrapper *)readWithPartition:(NSString *)partition
```

Comment:
We might need to adjust the signature a bit, or have an internal `MSDocumentWrapper` object in order to expose the pending operation.